ide/coqide/ideutils.ml

```diff
@@ -174,6 +174,19 @@ let ulen uni_ch =
   else if uni_ch < 0x10000 then 3
   else 4
 
+(* workaround for lablgtk bug up to version 2.18.13 *)
```

Comment:
The version number doesn't seem right to me folks.

Reply:
@herbelin?

Reply:
It's 𝜋:  3.1.4